platform/viewer/src/connectedComponents/ToolbarRow.js

```diff
@@ -110,6 +115,14 @@ class ToolbarRow extends Component {
       zIndex: 999,
     };
 
+    const downloadScreenShotContainerStyle = {
+      display: this.state.isDownloadScreenShotDialogOpen ? 'block' : 'none',
+      position: 'absolute',
+      top: '82px',
+      left: '600px',
```

Comment:
This may work on your monitor, but what about on a table or phone?

Reply:
@deprecated - We are using Modals now.